GitUI/CommandsDialogs/RevisionDiffControl.cs

```diff
@@ -189,6 +189,11 @@ bool SelectFirstGroupChangesIfFileNotFocused()
 
         public void ReloadHotkeys()
         {
+            if (!Visible)
+            {
+                return;
+            }
+
             LoadHotkeys(HotkeySettingsName);
```

Comment:
Why not do this inside `LoadHotkeys`?

Reply:
> Not formally approving as I am not sure how this affects key loading. For sure, not crashing reloading is better, the consequence should be to not load hotkeys OnLoad and HandleSettingsChanged.
Any alternatives?

> Why not do this inside LoadHotkeys?

Done. It took some time because of the misleading `TryGetUICommands`.